stdeb.cfg

```diff
@@ -0,0 +1,6 @@
+[colcon-meson]
+No-Python2:
+Depends3: python3-colcon-core (>= 0.10.0), python3-colcon-library-path
+Recommends3: meson, ninja-build
```

Comment:
Why is `meson` recommended rather than a required dependency? I would expect it to be required like `cmake` is in [colcon-cmake](https://github.com/colcon/colcon-cmake/blob/master/stdeb.cfg#L3).

Reply:
The extension requires at least meson version 0.60, which is only available in Ubuntu 22.04. Other distributions need to install a recent version via pip.

If I require meson here, all these distributions will have a broken `colcon-meson` installation (it will fail at the next meson API call). Alternatively, we could release this only to Ubuntu 22.04

Reply:
I think it does make sense to only release the deb where its dependencies are available via debs. Which in this case means no focal, buster, or bullseye. I think it also makes sense to express the meson version dependency in the Debian data as well as in setup.cfg.

I've pushed a couple of commits which implement these changes but feel free to rewrite or drop them if you prefer something different.